client/state/sites/hooks/use-premium-global-styles.ts

```diff
@@ -41,34 +38,18 @@ const getGlobalStylesInfoForSite = ( siteId: number | null ): GlobalStylesStatus
 		.catch( () => DEFAULT_GLOBAL_STYLES_INFO );
 };
 
-export function usePremiumGlobalStyles(): GlobalStylesStatus {
-	const params = new URLSearchParams( window.location.search );
-	const siteSlugParam = params.get( 'siteSlug' );
-	const siteIdParam = params.get( 'siteId' );
-	const selectedSiteId = useSelector( getSelectedSiteId );
-	const onboard = select( ONBOARD_STORE ).getState();
-
-	// When site id is null it means that the site hasn't been created yet.
```

Comment:
Maybe we could add this comment to line 20, it's not immediately obvious that a null site id means that the site isn't created.